docs/advanced-topics/physics.md

```diff
@@ -4,30 +4,40 @@ title: Physics
 
 Some collision events aren't fired when using `NetworkRigidbody`.
-- On the `server` all collision and trigger events (such as OnCollisionEnter) will fire as expected and you can access / modify values of the `Rigidbody` such as velocity.
-- On the `clients` the rigidbody will be kinematic. Trigger events will still be fired but collision events won't be fired when colliding with other networked rigidbodies.
+- On the `server`, all collision and trigger events (such as `OnCollisionEnter`) fire as expected and you can access (and change) values of the `Rigidbody` (such as velocity).
+- On the `clients`, the `Rigidbody` is kinematic. Trigger events still fire but collision events won't fire when colliding with other networked `Rigidbody` instances.
 
 :::tip
-If there is a need for a gameplay event to happen on a collision you can listen to `OnCollisionEnter` function on the server and synchronize the event via ClientRpc to all clients.
+
+If there is a need for a gameplay event to happen on a collision, you can listen to `OnCollisionEnter` function on the server and synchronize the event via `ClientRpc` to all clients.
+
 :::
 
 ## NetworkRigidbody2D
 
-`NetworkRigidbody2D` works in the same way as `NetworkRigidbody` but for 2d physics (`Rigidbody2D`) instead.
+`NetworkRigidbody2D` works in the same way as `NetworkRigidbody` but for 2D physics (`Rigidbody2D`) instead.
 
 ## NetworkRigidbody & ClientNetworkTransform
 
-`NetworkRigidbody` can be used in combination with the [ClientNetworkTransform](../components/networktransform.md#clientnetworktransform) package sample to allow the owner client of a `NetworkObject` to move it authoritatively. In this mode collisions will only result in realistic dynamic collisions if the object is colliding with other `NetworkObjects` which are owned by the same client.
+You can use `NetworkRigidbody` with the [`ClientNetworkTransform`](../components/networktransform.md#clientnetworktransform) package sample to allow the owner client of a `NetworkObject` to move it authoritatively. In this mode, collisions only result in realistic dynamic collisions if the object is colliding with other `NetworkObjects` (owned by the same client).
 
 :::note
-Add the `ClientNetworkTransform` component first to your `GameObject` else the `NetworkRigidbody` will automatically add a regular `NetworkTransform`.
-:::
\ No newline at end of file
+
+Add the `ClientNetworkTransform` component to your `GameObject` first. Otherwise the `NetworkRigidbody` automatically adds a regular `NetworkTransform`.
+
+:::
+
+## Physics and latency
+
+A common issue with physics in multiplayer games is lag and how objects update on basically different timelines. For example, a player would be on a timeline that’s offset by the network latency relative to your server’s objects. One way to prepare for this is to test your game with artificial lag. You might catch some see weird delayed collisions that would otherwise make it into production.
```

Comment:
```suggestion
A common issue with physics in multiplayer games is lag and how objects update on basically different timelines. For example, a player would be on a timeline that’s offset by the network latency relative to your server’s objects. One way to prepare for this is to test your game with artificial lag. You might catch some weird delayed collisions that would otherwise make it into production.
```